extensions/Cake.Parallel.Module.yml

```diff
@@ -0,0 +1,16 @@
+Type: Module
+Name: Cake.Parallel.Module
+NuGet: Cake.Parallel.Module
+Assemblies:
+- /**/*.dll
+Repository: 
```

Comment:
Repo: https://github.com/aabenoja/Cake.Parallel

Reply:
Manually updating the repository URL in the YAML file is not a practical option because it gets overwritten every time the discoverer analyses an addin. Better solution would be to raise an issue in the addin repo to encourage the author to update their nuspec.